### PR TITLE
Steps to "create a tag"

### PR DESCRIPTION
**Pull request by aaradhak**
Documentation for creating a tag as part of releasing a new version of the package.

### CODE REVIEW

File: RELEASE.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Craft a commit message for the tag.
+LASTTAG='v0.21.2'
+VERSION='0.21.3'
+NEWTAG="v${VERSION}"
+cat <
```
Comment (dustymabe) [inline RELEASE.txt:5]:
```suggestion
cat <<EOF > release-notes.txt
```

File: RELEASE.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+This is release ${NEWTAG} of console-login-helper-messages.
+
+Thanks to the following contributors for patches during this release: 
+
+$(git shortlog --no-merges --pretty=format:"%h %s" -e ${LASTTAG}..HEAD)
+EOF
+
+# After crafting message then create the tag
+
```
Comment (dustymabe) [inline RELEASE.txt:14]:
```suggestion
```

Delete this line

File: RELEASE.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+git tag --file release-notes.txt -s ${NEWTAG}
+
+# After creating the tag view it and verify signature:
```
Comment (dustymabe) [inline RELEASE.txt:17]:
```suggestion
# After creating the tag view it and verify signature:
git show ${NEWTAG}
git tag --verify ${NEWTAG}
```

**Comment by travier** [comment]
This should probably be done via https://github.com/coreos/repo-templates ?

File: RELEASE.txt (hunk 2 above)
Comment (dustymabe) [reply RELEASE.txt:14]: still need to delete this line

File: RELEASE.txt [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# and push it to the remote (not a fork)
```
Comment (dustymabe) [inline RELEASE.txt:18]: this comment should be moved down to just above the `git push --tags` line

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by dustymabe** [comment]
> This should probably be done via https://github.com/coreos/repo-templates ?

Hmm. I really don't know. This just shows someone the steps for creating/pushing a tag with some recent commit history info. Should that go in `repo-templates`?

**Comment by bgilbert** [comment]
> Hmm. I really don't know. This just shows someone the steps for creating/pushing a tag with some recent commit history info. Should that go in `repo-templates`?

I could see this going either way.  I think it's okay to keep one-off checklists  in their respective repositories; we can always move them later.  But if you're feeling ambitious, there would be value in a lightweight issue template for tagging a release.  [stream-metadata-go](https://github.com/coreos/stream-metadata-go) and [fedora-coreos-stream-generator](https://github.com/coreos/fedora-coreos-stream-generator/) could also use it.

**Comment by travier** [comment]
We will need to make packaging updates for Fedora & C9S going forward for CLHM so I think the template would be useful here.

**Comment by aaradhak** [comment]
@travier  I will merge this PR here

**Comment by travier** [comment]
OK, my previous message was ambiguous. I'd prefer we setup the templates for this repo using https://github.com/coreos/repo-templates as we will need the parts for packaging instead of having a ad-hoc one here.

**Pull request merged**